src/components/dropdown/Dropdown.vue

```diff
@@ -268,6 +322,21 @@ export default {
         close() {
             this.filteredItems = this.items;
             this.toggle = false;
+        },
+        addItem() {
+            const itemExists = this.items.some(item => item.text.toLowerCase() === this.itemToAdd.toLowerCase());
```

Comment:
You don't need to check again if `!itemExists` as the `addIcon` can only be shown if it doesn't exist in the `onFilterChange` function